client/packages/programs/src/JsonForms/components/ProgramSearch.tsx

```diff
@@ -62,6 +77,18 @@ const UIComponent = (props: ControlProps) => {
 
   if (zErrors) return <Typography color="error">{zErrors}</Typography>;
 
+  const allProgramsOptionRenderer: AllOptionsType = {
+    id: 'AllProgramsSelector',
+    name: t('label.all-programs'),
+  };
+
+  const programs = data?.nodes ?? [];
+
+  const programOptions =
+    programs.length > 1 && props.uischema.options?.['allProgramsOption']
```

Comment:
If you only have 1 program then that's the only one you select correct? 

```suggestion
    programs.length > 2 && props.uischema.options?.['allProgramsOption']
```